src/Workspaces/Core/Portable/CodeActions/Operations/ApplyChangesOperation.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace Microsoft.CodeAnalysis.CodeActions
 {
+#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
💭 We should probably fix RS0030 to not report diagnostics in comments.

Reply:
agreed :)

Reply:
I think I'd disagree here -- this is correctly pointing to a API smell. We have a documentation comment telling people to use APIs that we know are broken, but aren't providing a proper replacement for.